andela/Lit-Kigali1-Frontend
# #165020870 Restructure folder structure

#### What does this PR do?
Restructure folder structure

#### Description of Task to be completed?
- Have folder strucutre restructured
- Have eslint configured
- Have enzyme js configured
- Have one tests running
 
#### How should this be manually tested?
- Clone the repo
- Run command `npm run install` or `yarn` to install dependencies
- Run command  `npm run lint` or `yarn lint` to see linting issues
- Run command `npm run lint:fix` or `yarn lint:fix` to fix linting issues
- Run command `npm run test` or `yarn test` to run tests

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#165020870

#### Screenshots (if appropriate)
N/A

#### Questions
N/A
